### PR TITLE
Add Gerrit SVG for code host list

### PR DESCRIPTION
**Pull request by pjlast**
Adds a custom Gerrit SVG Icon since `mdi-react` doesn't have one:

Screenshot 2023-01-31 at 12 14 22

Also works with dark mode:

Screenshot 2023-01-31 at 12 20 49

## Test plan

Did a manual visual test. See screenshot

## App preview:

- [Web](https://sg-web-pjlast-gerrit-icon.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.45 kb) | 0.00% (+0.45 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/pjlast/gerrit-icon/compare-report.html;tab=live_object) for a full comparison between the commits e9978cf and 049d598 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/components/externalServices/externalServices.tsx [hunk 1]
```diff
@@ -1257,10 +1258,29 @@ const PAGURE: AddExternalServiceOptions = {
...
+            d="M4,0 h32 a4,4 0 0 1 4,4 v8 h8 a4,4 0 0 1 4,4 v32 a4,4 0 0 1 -4,4 h-32 a4,4 0 0 1 -4,-4 v-8 h-8 a4,4 0 0 1 -4,-4 v-32 a4,4 0 0 1 4,-4 z
+         m14,22 h12 v4 h-12 v-4 z
+         m16,0 h12 v4 h-12 v-4 z
+         m-16,14 h4 v-4 h4 v4 h4 v4 h-4 v4 h-4 v-4 h-4 v-4 z
+         m16,0 h4 v-4 h4 v4 h4 v4 h-4 v4 h-4 v-4 h-4 v-4 z"
+            fillRule="evenodd"
+        />
+    </svg>
+)
+
```
Comment (kopancek) [inline client/web/src/components/externalServices/externalServices.tsx:1279]: Nice inlining 👍 

Can we put it into it's own file though and export it from there?

**Review by kopancek — CHANGES_REQUESTED** [review]

File: client/web/src/components/externalServices/externalServices.tsx (hunk 1 above)
Comment (mrnugget) [reply client/web/src/components/externalServices/externalServices.tsx:1279]: Yeah, seems like icons-in-separate-files is the way to go: https://sourcegraph.sourcegraph.com/search?q=context:sourcegraph+%3Csvg+file:client/web+-file:snap%24&patternType=standard&sm=0&groupBy=repo

**Review by mrnugget — APPROVED** [review]
VERY NICE!!

**Review by sashaostrikov — APPROVED** [review]
:shipit:     :shipit:  :shipit:  :shipit::shipit::shipit::shipit:  :shipit::shipit::shipit::shipit:
:shipit::shipit:  :shipit:  :shipit:  :shipit:       :shipit:   
:shipit: :shipit: :shipit:  :shipit:  :shipit:       :shipit::shipit::shipit: 
:shipit:  :shipit::shipit:  :shipit:  :shipit:       :shipit:    
:shipit:     :shipit:  :shipit:  :shipit::shipit::shipit::shipit:  :shipit::shipit::shipit::shipit:

**Review by kopancek — APPROVED** [review]
```          _____                    _____                    _____                    _____          
         /\    \                  /\    \                  /\    \                  /\    \         
        /::\____\                /::\    \                /::\    \                /::\    \        
       /::::|   |                \:::\    \              /::::\    \              /::::\    \       
      /:::::|   |                 \:::\    \            /::::::\    \            /::::::\    \      
     /::::::|   |                  \:::\    \          /:::/\:::\    \          /:::/\:::\    \     
    /:::/|::|   |                   \:::\    \        /:::/  \:::\    \        /:::/__\:::\    \    
   /:::/ |::|   |                   /::::\    \      /:::/    \:::\    \      /::::\   \:::\    \   
  /:::/  |::|   | _____    ____    /::::::\    \    /:::/    / \:::\    \    /::::::\   \:::\    \  
 /:::/   |::|   |/\    \  /\   \  /:::/\:::\    \  /:::/    /   \:::\    \  /:::/\:::\   \:::\    \ 
/:: /    |::|   /::\____\/::\   \/:::/  \:::\____\/:::/____/     \:::\____\/:::/__\:::\   \:::\____\
\::/    /|::|  /:::/    /\:::\  /:::/    \::/    /\:::\    \      \::/    /\:::\   \:::\   \::/    /
 \/____/ |::| /:::/    /  \:::\/:::/    / \/____/  \:::\    \      \/____/  \:::\   \:::\   \/____/ 
         |::|/:::/    /    \::::::/    /            \:::\    \               \:::\   \:::\    \     
         |::::::/    /      \::::/____/              \:::\    \               \:::\   \:::\____\    
         |:::::/    /        \:::\    \               \:::\    \               \:::\   \::/    /    
         |::::/    /          \:::\    \               \:::\    \               \:::\   \/____/     
         /:::/    /            \:::\    \               \:::\    \               \:::\    \         
        /:::/    /              \:::\____\               \:::\____\               \:::\____\        
        \::/    /                \::/    /                \::/    /                \::/    /        
         \/____/                  \/____/                  \/____/                  \/____/

**Pull request merged**